system-tests/e2e-transfer-test/resources/kafka/docker-compose.yaml

```diff
@@ -0,0 +1,26 @@
+version: '3.8'
+
+services:
+
+  zookeeper:
+    image: wurstmeister/zookeeper:3.4.6
```

Comment:
is there a particular reason you are not using the [official zookeeper image](https://hub.docker.com/_/zookeeper) and the [Bitnami Kafka image](https://hub.docker.com/r/bitnami/kafka)?

Reply:
Didn't Kafka also deprecate Zookeeper and replace it with their own Raft implementation?

Reply:
Replaced with KRaft